lib/application/completions/completions_state.dart

```diff
@@ -6,20 +6,25 @@ class CompletionsState extends Equatable {
   final String id;
   final Responses responses;
   final bool isComplete;
+  final UploadTask uploadTask;
+  final Map<String, String> downloadURL;
```

Comment:
What would happen if there are multiple `image_upload` body types? Is `CompletionsState` only able to handle a single image URL here?